LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/resources/fetch-event-test-worker.js

```diff
@@ -47,6 +47,16 @@ function handleResultingClientId(event) {
   event.respondWith(new Response(body));
 }
 
+function handleReplacesClientId(event) {
+  var body;
+  if (event.replacesClientId !== "") {
+    body = 'Replaces Client ID Found: ' + event.replacesClientId;
```

Comment:
For same origin clients, we should be able to get the Client from its id and do some checks.

Reply:
Ok, will look into it.